[intel-mkl] build failure using GitHub Actions (MSBuild based project)

**Host Environment**
- OS: Windows via GitHub Action ([msvc_build.yml](https://github.com/rogerfraser/DynAdjust/blob/master/.github/workflows/msvc_build.yml))
  ```yaml
  env:
    BUILD_CONFIGURATION: Release
    ARCH: x64

  jobs:
    build:
      runs-on: windows-latest
  ...
  
  ```
  
- Compiler: MSBuild (supplied by GitHub Actions virtual environments)
  ```yaml
  steps:
  ...

  - name: Add MSBuild to PATH
    uses: microsoft/setup-msbuild@v1.0.2  
    
  ...
  
  ```

**To Reproduce**
Steps to reproduce the behavior:
```yaml
- name: Install vcpkg prerequisites
  shell: cmd
  if: steps.cache.outputs.cache-hit != 'true'
  run: |
    vcpkg update
    vcpkg install intel-mkl --triplet=${{env.ARCH}}-windows
    vcpkg ...
    vcpkg integrate install
    vcpkg list
    
  ...
  
  ```

**Failure logs** 
- See full log [here](https://github.com/rogerfraser/DynAdjust/runs/3034571347?check_suite_focus=true).

```txt
Run vcpkg update
  vcpkg update
  vcpkg install intel-mkl --triplet=x64-windows
  shell: C:\Windows\system32\cmd.EXE /D /E:ON /V:OFF /S /C "CALL "{0}""
  env:
    SOLUTION_FILE_PATH: ./dynadjust/dynadjust_1_01_00.sln
    BUILD_CONFIGURATION: Release
    ARCH: x64
    VCPKG_INSTALL_FOLDER: c:/vcpkg/installed
    CS_XSD_INSTALL_FOLDER: xsd/xsd-4.0.0-i686-windows/libxsd
Using local portfile versions. To update the local portfiles, use `git pull`.
No packages need updating.
Computing installation plan...
The following packages will be built and installed:
    intel-mkl[core]:x64-windows -> 2020.0.0
Detecting compiler hash for triplet x64-windows...
Could not locate cached archive: C:\Users\runneradmin\AppData\Local\vcpkg\archives\f8\f8e99c4db6108860ec37bb79c9bbdbbc05a34887cc655ec8fcd3a7400f38d72b.zip
Starting package 1/1: intel-mkl:x64-windows
Building package intel-mkl[core]:x64-windows...
CMake Error at ports/intel-mkl/portfile.cmake:15 (message):
  Could not find MKL.  Before continuing, please download and install MKL
  (20200000 or higher) from:

      https://registrationcenter.intel.com/en/products/download/3178/

  

  Also ensure vcpkg has been rebuilt with the latest version (v0.0.104 or
  later)
Call Stack (most recent call first):
  scripts/ports.cmake:139 (include)


Error: Building package intel-mkl:x64-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: intel-mkl:x64-windows
  Vcpkg version: 2021-05-05-b110959e13f562c2a17d7bd9e352e2dd8d6dc603

Additionally, attach any relevant sections from the log files above.
Error: Process completed with exit code 1.
```

**Additional context**
- Successful build of the repo requires other packages (e.g. boost-*, 7zip, xerces-c, etc.).   Installation of these packages (e.g. `vcpkg install boost-process ... --triplet=${{env.ARCH}}-windows`) is successful. 
- Caching was originally applied, then removed, but neither cases were successful.
- Full yaml [here](https://github.com/rogerfraser/DynAdjust/blob/master/.github/workflows/msvc_build.yml).

What is the question? 
```
CMake Error at ports/intel-mkl/portfile.cmake:15 (message):
  Could not find MKL.  Before continuing, please download and install MKL
  (20200000 or higher) from:

      https://registrationcenter.intel.com/en/products/download/3178/

```

--- a/ports/intel-mkl/CONTROL
+++ b/ports/intel-mkl/CONTROL
@@ -1,4 +1,0 @@
-Source: intel-mkl
-Version: 2020.0.0
-Port-Version: 1
-Description: Intel® Math Kernel Library (Intel® MKL) accelerates math processing routines, increases application performance, and reduces development time on Intel® processors. 

--- a/ports/intel-mkl/vcpkg.json
+++ b/ports/intel-mkl/vcpkg.json
@@ -1,0 +1,6 @@
+{
+  "name": "intel-mkl",
+  "version": "2020.0.0",
+  "port-version": 2,
+  "description": "Intel® Math Kernel Library (Intel® MKL) accelerates math processing routines, increases application performance, and reduces development time on Intel® processors. Note that please download and install MKL from https://registrationcenter.intel.com/en/products/download/3178/ manually before using it in vcpkg."
+}

--- a/ports/intel-mkl/vcpkg.json
+++ b/ports/intel-mkl/vcpkg.json
@@ -2,5 +2,5 @@
   "name": "intel-mkl",
   "version": "2020.0.0",
   "port-version": 2,
-  "description": "Intel® Math Kernel Library (Intel® MKL) accelerates math processing routines, increases application performance, and reduces development time on Intel® processors. Note that please download and install MKL from https://registrationcenter.intel.com/en/products/download/3178/ manually before using it in vcpkg."
+  "description": "Intel® Math Kernel Library (Intel® MKL) accelerates math processing routines, increases application performance, and reduces development time on Intel® processors. Due to the complexity involved, this package doesn't install MKL. It instead verifies that MKL is installed. Please download and install MKL from https://registrationcenter.intel.com/en/products/download/3178/ manually before using it in vcpkg."
 }

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2726,7 +2726,7 @@
     },
     "intel-mkl": {
       "baseline": "2020.0.0",
-      "port-version": 1
+      "port-version": 2
     },
     "intelrdfpmathlib": {
       "baseline": "20U2-1",

--- a/versions/i-/intel-mkl.json
+++ b/versions/i-/intel-mkl.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "88fbc68eed77db424c50b2d4796a6478524ba0c7",
+      "version": "2020.0.0",
+      "port-version": 2
+    },
+    {
       "git-tree": "1f91c30839fe3222faa4c81d978ae4f52f61a1df",
       "version-string": "2020.0.0",
       "port-version": 1

--- a/versions/i-/intel-mkl.json
+++ b/versions/i-/intel-mkl.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "88fbc68eed77db424c50b2d4796a6478524ba0c7",
+      "git-tree": "f2d5b828df130d8b1106aa0c076eb9cb79e50ea9",
       "version": "2020.0.0",
       "port-version": 2
     },

